### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dmytropodelnik**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -50,3 +50,31 @@ Testnet tx: [link](https://testnet.bscscan.com/tx/0x298d3484e9532ecddabf3d2d578f
...
+```
+Testnet tx: [link](https://rinkeby.etherscan.io/tx/0x7eeea83ca80c654cf59c9155db2991ce41298cf266d14b90e111ab6b6cbce682)
+
+If you want to transfer initiating ownership address to another you should use InitiateTransferOwnership() function.
+```solidity
+    function InitiateTransferOwnership(address target) external;
+```
+Testnet tx: [link](https://rinkeby.etherscan.io/tx/0xa07d1b1a4bc5939f1df0b6ed5e9c4ccfcb1ea1159b0155c11e8ccc388e5cd623)
+
+If you want to confirm initiated transfer ownership address to another you should use ConfirmTransferOwnership() function.
```
Comment (YouStillAlive) [inline README.md:76]: Could change to: After that, you need to collect the required number of votes. The ConfirmTransferOwnership() function allows members to vote. If the required number of votes is obtained, the rights will be transferred.

File: README.md [hunk 2]
```diff
@@ -50,3 +50,31 @@ Testnet tx: [link](https://testnet.bscscan.com/tx/0x298d3484e9532ecddabf3d2d578f
...
+```
+Testnet tx: [link](https://rinkeby.etherscan.io/tx/0xaa07b87cb97a1d6c24d52fb00b445a6d5d0805aed1d8c6375ef2e6955c92ced3)
+
+If you want to change your authorized address to another you should use ChangeAuthorizedAddress() function.
+```solidity
+    function ChangeAuthorizedAddress(address authorize) external;
+```
+Testnet tx: [link](https://rinkeby.etherscan.io/tx/0x7eeea83ca80c654cf59c9155db2991ce41298cf266d14b90e111ab6b6cbce682)
+
+If you want to transfer initiating ownership address to another you should use InitiateTransferOwnership() function.
```
Comment (YouStillAlive) [inline README.md:70]: Could delete "initiating" word

File: README.md (hunk 1 above)
Comment (dmytropodelnik) [reply README.md:76]: yep, understood!)

File: README.md (hunk 2 above)
Comment (dmytropodelnik) [reply README.md]: ok

**Review by YouStillAlive — APPROVED** [review]

**Pull request merged**